tests/test_dump_file.py

```diff
@@ -187,3 +187,71 @@ def test_copy_from(snap, tmp_path):
     assert numpy.all(read_snap.molecule == [0, 1, 2])
     assert read_snap.has_charge()
     assert numpy.allclose(read_snap.charge, [0, 1, -1])
+
+
+def test_faulty_dump_schema(snap, tmp_path):
+    # test image schema
```

Comment:
This should generate a 1d or 2d list of zeros based on how many entries are in the schema above.
```suggestion
    # test schema
    schema_zeros = [0] * len(schema.split())
```